docs/issue_template.md

```diff
@@ -1,41 +0,0 @@
-<!--
```

Comment:
Why are we removing this?

Reply:
good catch, this is one of the things i missed when doing the merge conflicts